CMSgov/design-system
# [WNMGDS-1721] Storybook Consolidation with Theme Capabilities for SB & Gatsby

Initial PR, don't merge until sprint 3.3. Will be adding loki updates.

## Summary

- Consolidates Storybook into a single themable instance! Still buildable separately. There will be a future ticket in 3.3 added to this to show/hide extraneous stories based on environment variable, this would help with loki testing issues
- A few small style fixes to stories
- Adds theme switching capability to both storybook & gatsby via query param, localStorage & environment variables
- Adds medicare fonts to gatsby / storybook

## Questions

- Should `localStorage` be swapped out for `sessionStorage`?

## How to test

- Make sure you're on node 16.
- `yarn install && yarn build-storybook` `http-server storybook-static` to check out a static build, can also `yarn build-storybook:healthcare` etc.. to see the static build of the other sites that way, check out the theme switching option.. when you build a child system you should see that systems theme selected by default, look at a component like monthpicker and note that the inverse background is set correctly
- `yarn storybook` will give you the same thing as above
- `yarn build-storybook:gatsby && yarn build:gatsby && yarn start:gatsby` - add `?theme=medicare` `?theme=healthcare` to the url when loaded, note localStorage item updates. check out pages with iframes and note a theme change updates the iframe content
-
